app/src/ui/lib/dark-theme.ts

```diff
@@ -4,7 +4,10 @@ import {
   isWindows10And1809Preview17666OrLater,
 } from '../../lib/get-os'
 
-export function supportsDarkMode() {
+/**
```

Comment:
I'm not sure if this file should be renamed to something like `system-theme.ts` as well

Reply:
part of me wants to merge this file into `app/src/ui/lib/application-theme.ts`

Reply:
Agree. It does feel arbitrarily separated from application-theme given that supportsDarkMode just became more generic.

Reply:
Sounds good to me!